utils/firestore.js

```diff
@@ -1909,21 +1909,36 @@ const sendClientEmail = async (data) => {
         read: data.read
     }
 
-    await storeNotifications(reminder);
+    await storeNotification(reminder);
 
     sendEmail(data.email, data.subject, data.message);
     return true;
 };
 
-const storeNotifications = async payload => {
+const storeNotification = async (notificationData) => {
     try {
-        await db.collection('notifications').add(payload);
+        await db.collection('notifications').add(notificationData);
     } catch (error) {
         console.error(error);
         return new Error(error);
     }
 }
 
+/**
+ * 
+ * @param {string} userToken User token
+ * @param {string} specId Notification Specifications ID
```

Comment:
NIT: Should we define `return` in DocBlock

Reply:
The return type is correctly inferred as `Promise<boolean>` by editor, so I didn't repeat its work.